highcharts-behavior.html

```diff
@@ -35,6 +35,7 @@
             yZoom: {type: Boolean, value: false, reflectToAttribute: true, observer: "_zoomUpdate"},
             loading: {type: Boolean, value: false, observer: "_loadingUpdate", reflectToAttribute:true},
             loadingMessage: {type: String, value: ""},
+            renderer: {type: Object, computed: "_renderer(_chart)"},
```

Comment:
You might just be able to do:
```javascript
renderer: {type: Object, computed: "get(_chart.renderer)"},
```

This way, as the renderer changes (but not the `_chart` object, ex. during initialization), the renderer is automatically binded, without supporting functions.